app/controllers/api/account.php

```diff
@@ -3077,3 +3166,82 @@
 
         $response->dynamic($verificationDocument, Response::MODEL_TOKEN);
     });
+
+App::put('/v1/account/targets/:targetId/push')
+    ->desc('Update Account Target')
+    ->groups(['api', 'account'])
+    ->label('error', __DIR__ . '/../../views/general/error.phtml')
+    ->label('audits.event', 'target.update')
+    ->label('audits.resource', 'target/response.$id')
+    ->label('event', 'users.[userId].targets.[targetId].create')
+    ->label('scope', 'public')
```

Comment:
Let's add the sdk labels here too and remove public scope

Reply:
removed these